client/src/Movies/SavedList.js

```diff
@@ -10,9 +11,13 @@ export default class SavedList extends Component {
       <div className="saved-list">
         <h3>Saved Movies:</h3>
         {this.props.list.map(movie => (
-          <span className="saved-movie">{movie.title}</span>
+          <NavLink to={`/movies/${movie.id}`}>
```

Comment:
In React I never want you to forget the rule of thumb after using the map method
You Ought to place a unique key on the first element after mapping
```suggestion
          <NavLink to={`/movies/${movie.id}`} key={movie.id + Date.now()}>
```

Reply:
we are using date.now because nothing is stopping us from adding the same movie twice which means there will be two keys with the same value